source/scpp/src/xdr/fix-stellar-scp-public.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/source/scpp/src/xdr/Stellar-SCP.h b/source/scpp/src/xdr/Stellar-SCP.h
+index 6f6f4d52..8755b4fc 100644
+--- a/source/scpp/src/xdr/Stellar-SCP.h
++++ b/source/scpp/src/xdr/Stellar-SCP.h
+@@ -234,7 +234,7 @@ struct SCPStatement {
+     };
+ 
+     using _xdr_case_type = xdr::xdr_traits<SCPStatementType>::case_type;
+-  private:
```

Comment:
The comment `/*  private:*/  // BPFK note: cannot be private as we require runtime layout checks` was lost in the process. It would be good to keep it here so other people know why it was changed to public.